coeus-impl/src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java

```diff
@@ -79,15 +83,28 @@
         }
 
         boolean usePrimaryKeys = getLookupService().allPrimaryKeyValuesPresentAndNotWildcard(Award.class, fieldValues);
-        
+
         setBackLocation(fieldValues.get(KRADConstants.BACK_LOCATION));
         setDocFormKey(fieldValues.get(KRADConstants.DOC_FORM_KEY));
         setReferencesToRefresh(fieldValues.get(KRADConstants.REFERENCES_TO_REFRESH));
-        
+
+        if (StringUtils.isNotBlank(fieldValues.get(IP_NUMBER_FIELD))) {
+            String awardNumbers =
+                    getBusinessObjectService().findMatching(AwardFundingProposal.class, Collections.singletonMap("proposal.proposalNumber", fieldValues.get(IP_NUMBER_FIELD)))
+                            .stream()
+                            .map(afp -> afp.getAward().getAwardNumber())
+                            .collect(Collectors.joining("|"));
+            if (StringUtils.isBlank(awardNumbers)) {
+                return new ArrayList<>();
+            }
+            fieldValues.put("awardNumber", awardNumbers);
```

Comment:
If awardNumber is set as a search criteria then append.  Don't just overwrite the value with the map.put